spec/models/availability_spec.rb

```diff
@@ -34,4 +36,31 @@
       expect(availability.partially_available?(event)).to eq(false)
     end
   end
+
+  before(:each) do
+    availability = build(:availability, person: a_person, start_time: start_time, end_time: end_time)
+    availability.save
```

Comment:
If you did create in line 41, would you need the save ? 
Also see @david135 comments on build_stubbed.